subprojects/core-api/src/main/java/org/gradle/StartParameter.java

```diff
@@ -59,7 +57,6 @@
  *
  * <p>You can obtain an instance of a {@code StartParameter} by either creating a new one, or duplicating an existing one using {@link #newInstance} or {@link #newBuild}.</p>
  */
-@ServiceScope(Scopes.BuildSession.class)
```

Comment:
What's this change about?

Reply:
This is so the configuration cache will serialize `StartParameter` instances as regular beans instead of service references. It fixes `GradleBuildTaskIntegrationTest`.